### PR TITLE
Updates to sync schema with specification

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
Adjustments to correct differences between the schema and the specification documentation.

### CODE REVIEW

File: specification/objects/action.md [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
-- **affectsContext** (one): The conceptual entity where the impacts are realized from successful exploitation of a security vulnerability. Different impacts can be realized by multiple contexts from multiple scenarios. (See [Context](../values/context.md)) <br />
-- **hasEntityRole** (one): Describes the role an associated Context performs in the Vulnerability Scenario Action being described. (See [Entity Role](../values/entity-role.md)) <br />
 
+- **id** (one): A globally unique identifier for the action that distinguishes it from other actions related to the same vulnerability.
+- **hasName** (zero or one): A name or label to assist in identifying a given action in the context of the containing Vulnerability. This name should be unique across all sibling actions.
+- **hasImpactMethod** (one or many): A description of the method used to exploit a vulnerability providing some additional information on the impact of exploitation. (See [Impact Method](../values/impact-method.md))
```
Comment (david-waltermire-nist) [inline specification/objects/action.md:9]:
```suggestion
- **hasImpactMethod** (one or many): A description of the method used to exploit a vulnerability providing some additional information on the impact of exploitation. (See [Impact Method](../object/impact-method.md))
```

File: specification/objects/barrier.md [hunk 2]
```diff
@@ -2,39 +2,36 @@
...
-  - **Boundary Protections**:  The attacker must be able to move through explicit boundaries that otherwise would prevent successful exploitation
-  
-	*Type Specific Properties* <br />
-  **relatesToContext** (one): The context to which the boudary is related (See [Context](../values/context.md)).
-    - **Sandbox**:  The product is deployed within a sandbox that must be broken out from to recognize impacts
-    - **Container**:  The product is deployed within a type of container that must be broken out from to recognize impacts.
+## Properties
+
+- **id** (one): A globally unique identifier for the impact that distinguishes it from other impacts related to the same vulnerability.
+- **hasBarrierType** (one): Identifies the kind of barrier. Based on the barrier's type, [additional properties](#types-of-barrier) may be required.
```
Comment (david-waltermire-nist) [inline specification/objects/barrier.md:8]:
```suggestion
- **hasBarrierType** (one): Identifies the kind of barrier. Based on the barrier's type, [additional properties](#additional-properties) may be required.
```

File: specification/objects/impact-method.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Impact Method Object
+
+A description of the method used to exploit a vulnerability providing some additional information on the impact of exploitation. These are intended to be high level concepts that lead to more granular impacts as referenced in the logical and physical impact entities.
+
+## Properties
+
+- **hasImpactMethodType** (one): The nature of impact.  See [Impact Method Types](../values/impact-method-type.md).
+
+- **hasGainedPrivilege** (one - see notes): Abstraction to assist in capturing relative privilege levels. The abstraction is only for the sake of discussing the vulnerability and is not intended to communicate the actual granular privileges that exist in most information system environments. See [Privilege Level](privilege-level-type.md)
```
Comment (david-waltermire-nist) [inline specification/objects/impact-method.md:9]:
```suggestion
- **hasGainedPrivilege** (one - see notes): Abstraction to assist in capturing relative privilege levels. The abstraction is only for the sake of discussing the vulnerability and is not intended to communicate the actual granular privileges that exist in most information system environments. See [Privilege Level](../values/privilege-level.md)
```

File: specification/objects/barrier.md [hunk 4]
```diff
@@ -2,39 +2,36 @@
...
-  - **Boundary Protections**:  The attacker must be able to move through explicit boundaries that otherwise would prevent successful exploitation
-  
-	*Type Specific Properties* <br />
-  **relatesToContext** (one): The context to which the boudary is related (See [Context](../values/context.md)).
-    - **Sandbox**:  The product is deployed within a sandbox that must be broken out from to recognize impacts
-    - **Container**:  The product is deployed within a type of container that must be broken out from to recognize impacts.
+## Properties
+
+- **id** (one): A globally unique identifier for the impact that distinguishes it from other impacts related to the same vulnerability.
+- **hasBarrierType** (one): Identifies the kind of barrier. Based on the barrier's type, [additional properties](#additional-properties) may be required.
```
Comment (Chris-Turner-NIST) [inline specification/objects/barrier.md:8]: Should add something similar to below, there is no connector to the new Types .md created.
(See [Barrier Types](../values/barrier-type.md))

**Review by Chris-Turner-NIST — APPROVED** [review]
LGTM

**Pull request merged**